### PR TITLE
added nix flake

### PR DESCRIPTION
**Pull request by hellwolf**

### CODE REVIEW

**Comment by simonmichael** [comment]
Input/alternatives from other nix users welcome.

File: flake.nix [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  let
+    pkgs = import nixpkgs { inherit system; };
+    hPkgs = pkgs.haskell.packages.ghc926; # need to match Stackage LTS version
+
+    myDevTools = [
+      pkgs.zlib
+      pkgs.libGL
+      pkgs.libGLU
+      pkgs.freeglut
+      hPkgs.ghc # GHC compiler in the desired version (will be available on PATH)
```
Comment (TristanCacqueray) [inline flake.nix:17]: Perhaps common libs could be provided with:

```suggestion
      (hPkgs.ghcWithPackages (p: [p.ansi-terminal-game p.gloss])) # GHC compiler in the desired version (will be available on PATH)
```

Comment (hellwolf) [reply flake.nix:17]: will try

Comment (hellwolf) [reply flake.nix]: updated

**Comment by simonmichael** [comment]
Thanks!

**Pull request merged**